.travis.yml

```diff
@@ -1,15 +1,19 @@
 language: python
 python:
   - "2.7"
-sudo: false
+sudo: true
 cache:
   # Workaround for
   # https://github.com/travis-ci/travis-ci/issues/3239
   directories:
     - ~/.cache/pip
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install swig
 before_script:
   - flake8 . --exclude=./docs/conf.py
 install:
+  - DEB_HOST_MULTIARCH=x86_64-linux-gnu pip install -I --exists-action=w "git+git://anonscm.debian.org/collab-maint/m2crypto.git@debian/0.21.1-3#egg=M2Crypto"
```

Comment:
Yeah, I was thinking that it probably won't be hard to port [encrypted cookies](https://github.com/brightinteractive/django-encrypted-cookie-session/) to cryptography